lessons/index.html

```diff
@@ -51,7 +51,8 @@ <h3>Javascript</h3>
     <li class="lesson"><a href="https://github.com/turingschool-examples/debugger-intro">Debugging Javascript</a></li>
     <li class="lesson"><a href="module-1/event-bubbling-and-delegation.html">Event Bubbling and Delegation</a></li>
     <li class="lesson"><a href="module-1/introduction-to-testing-javascript.html">Introduction to Unit Testing JavaScript</a></li>
-    <li class="lesson"><a href="module-1/array-prototype-methods-intro">Array Prototype Methods</a></li>
+    <li class="lesson"><a href="module-1/intro-to-array-prototype-methods.html">Intro to Array Prototype Methods</a></li>
+    <li class="lesson"><a href="module-1/array-prototype-methods-continued">Array Prototype Methods Continued</a></li>
```

Comment:
Could be good to call it 'array-prototype-iteration-methods'? That way they're building better vocabulary.

Reply:
Also, are you planning on streamlining that and pulling out some of the less essential pieces? (ie. lexicographical sorting)?

Reply:
Good call with the name.  Thanks!  Yes, I am planning on streamlining the iteration methods so it's a bit shorter and cutting out extras like the lexicographical sorting.